docs/HDRUK/3.0.0.md

```diff
@@ -17,9 +17,9 @@ Examples:
 
 Dataset metadata version.
         
-| title           | guidance   | is_list   | is_optional   | required   | type                                                                              |
-|:----------------|:-----------|:----------|:--------------|:-----------|:----------------------------------------------------------------------------------|
-| Dataset Version | N/A        | False     | False         | True       | ["Semver[{'pattern': '^([0-9]+)\\\\.([0-9]+)\\\\.([0-9]+)$', 'type': 'string'}]"] |
+| title           | guidance                                                                                                                                                                                                                                                          | is_list   | is_optional   | required   | type                                                                              |
```

Comment:
you were going to remove `is_optional` from the rendering?

Reply:
Yep, will be doing this one as it's own PR, need to mess around with the create_markdown function locally first